app/src/features/apiClient/store/apiRecords/Daemon/index.tsx

```
@@ -3,23 +3,24 @@ import { AutoSyncLocalStoreDaemon } from "features/apiClient/helpers/modules/syn
-  const isMulti = contexts.size > 1;
-  const daemons = Array.from(contexts.values()).map(({ id: contextId }) => (
-    <ContextId id={contextId}>
-      <EnvironmentDaemon />
-      <CollectionVariablesDaemon />
+  const contexts = apiClientContextRegistry.getAllContexts();
+  const isMulti = contexts.length > 1;
+          // <ExampleCollectionsDaemon />
+      // <EnvironmentDaemon />
+      // <CollectionVariablesDaemon />
+  const daemons = contexts.map(({ workspaceId: contextId }) => (
+    <WorkspaceProvider workspaceId={contextId}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)